reflex-dom-plaid/Reflex/Dom/Plaid/Link.hs

```diff
@@ -66,10 +68,8 @@ data PlaidLinkProduct
   deriving (Bounded, Enum, Eq, Ord, Show)
 
 data PlaidLinkConfig = PlaidLinkConfig
-  { _plaidLinkConfig_clientName :: !Text
-  , _plaidLinkConfig_env :: !PlaidLinkEnvironment
-  , _plaidLinkConfig_publicKey :: !Text
-  , _plaidLinkConfig_products :: ![PlaidLinkProduct]
```

Comment:
if `PlaidLinkProduct` and `PlaidLinkEnvironment` aren't used anywhere else they should be deleted.

Reply:
`PlaidLinkProduct` may become handy in future. And not sure about `PlaidLinkEnvironment`